statistics/handle/handle_hist.go

```diff
@@ -154,7 +155,7 @@ func (h *Handle) removeHistLoadedColumns(neededItems []model.TableItemID) []mode
 			continue
 		}
 		colHist, ok := tbl.Columns[item.ID]
-		if ok && colHist.IsStatsInitialized() && !colHist.IsFullLoad() {
+		if (!ok && tbl.ColAndIndexExistenceMap.HasAnalyzed(item.ID, false)) || (ok && colHist.IsStatsInitialized() && !colHist.IsFullLoad()) {
```

Comment:
Why do we need `HasAnalyzed` here? We are using `Has()` everywhere else.

Reply:
Other places just need to know whether this column exists or not.
Here should tell whether this column really has stats or not.

Reply:
I think the question is: if the col/idx stats are not in the `tbl.Columns`/`tbl.Indices`, what needs to be satisfied to trigger a loading? `Has()` or `HasAnalyzed()`?